react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+const images = [
+  {
+    name: 'Pikachu',
+    imgUrl: '../images/025.png'
+  },
+  {
+    name: 'Bulbasaur',
+    imgUrl: '../images/001.png'
+  },
+  {
+    name: 'Squirtle',
+    imgUrl: '../images/007.png'
+  },
+  {
+    name: 'Charmander',
+    imgUrl: '../images/004.png'
+  },
+  {
+    name: 'Jigglypuff',
+    imgUrl: '../images/039.png'
+  }
+];
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      count: 0
+    };
+    this.clickRight = this.clickRight.bind(this);
+    this.clickLeft = this.clickLeft.bind(this);
+    this.circleClick = this.circleClick.bind(this);
+    this.carouselInterval = this.carouselInterval.bind(this);
+  }
+
+  clickLeft() {
+    if (this.state.count === 0) {
+      this.setState({
+        count: 4
+      });
```

Comment:
Rather than assuming 5 images, use the length of your images prop to compute the correct bounds.